modules/cnf-performing-end-to-end-tests-for-platform-verification.adoc

```diff
@@ -721,25 +901,118 @@ More histogram entries ...
+# Avg Latencies: 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002 00002
+# Max Latencies: 00493 00387 00271 00619 00541 00513 00009 00389 00252 00215 00539 00498 00363 00204 00068 00520
+# Histogram Overflows: 00001 00001 00001 00002 00002 00001 00000 00001 00001 00001 00002 00001 00001 00001 00001 00002
+# Histogram Overflow at cycle number:
+# Thread 0: 155922
+# Thread 1: 110064
+# Thread 2: 110064
+# Thread 3: 110063 155921
+# Thread 4: 110063 155921
+# Thread 5: 155920
+# Thread 6:
+# Thread 7: 110062
+# Thread 8: 110062
+# Thread 9: 155919
+# Thread 10: 110061 155919
+# Thread 11: 155918
+# Thread 12: 155918
+# Thread 13: 110060
+# Thread 14: 110060
+# Thread 15: 110059 155917
+----
+
 [id="cnf-performing-end-to-end-tests-running-oslat"]
 === Running oslat
 
-[source, terminal]
+.Prerequisites
+
+* You logged into registry.redhat.io with your Customer Portal credentials
+* You installed the Performance Add-on Operator and you applied a performance profile
+
+.Procedure
+
+* To perform the `oslat`, run the following command: 
+
+[source,terminal,subs="attributes+"]
 ----
-$ docker run -v $(pwd)/:/kubeconfig -e KUBECONFIG=/kubeconfig/kubeconfig -e LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_CPUS=7 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20  registry.redhat.io/openshift4/cnf-tests-rhel8:v4.9 /usr/bin/test-run.sh -ginkgo.focus="oslat"
+$ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig -e \
+LATENCY_TEST_RUN=true -e DISCOVERY_MODE=true -e ROLE_WORKER_CNF=worker-cnf -e \
+LATENCY_TEST_CPUS=7 -e LATENCY_TEST_RUNTIME=600 -e MAXIMUM_LATENCY=20 \
+registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} /usr/bin/test-run.sh -ginkgo.focus="oslat"
 ----
 
-The above command runs the `cyclictest` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!`.
+The command runs the `oslat` tool for 10 minutes (600 seconds). The test runs successfully when the maximum observed latency is lower than `MAXIMUM_LATENCY` (20 μs), and the command line displays `SUCCESS!` when this test is completed.
+
+If the results exceed the latency threshold, the test fails and you can see the following output:
 
 .Example failure output
-[source, terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ docker run -v $KUBECONFIG:/root/kubeconfig:Z -e KUBECONFIG=/root/kubeconfig -e IMAGE_REGISTRY=quay.io/titzhak -e CNF_TESTS_IMAGE=cnf-tests:latest -e PERF_TEST_PROFILE=performance -e ROLE_WORKER_CNF=worker-cnf -e LATENCY_TEST_RUN=true -e LATENCY_TEST_RUNTIME=600 -e DISCOVERY_MODE=true -e MAXIMUM_LATENCY=20 -e LATENCY_TEST_CPUS=7 quay.io/titzhak/cnf-tests:latest usr/bin/test-run.sh -ginkgo.v -ginkgo.focus="oslat" <1>
+$ podman run -v $KUBECONFIG:/root/kubeconfig:Z -e KUBECONFIG=/root/kubeconfig -e \
```

Comment:
-v

>  -v $KUBECONFIG:/root/kubeconfig:Z -e KUBECONFIG=/root/kubeconfig
this is okay & works as needed. you may also preserve the same convention in all commands to avoid confusion, like:
`-v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig`

Reply:
@shajmakh I've updated the confusing instances